tasks/reduce-snapshot/tests/test-reduce-snapshot-missing-resource.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-reduce-snapshot-missing-resources
+spec:
+  description: |
+    Run the reduce task to reduce to a single component but CR to query is missing
```

Comment:
Can you say like "so it will look at the snapshot file instead"

Reply:
fixed description